src/statementsem.d

```diff
@@ -1448,7 +1448,16 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
                 else
                 {
                     if (global.params.safe)
-                        fld.tookAddressOf = 1;  // allocate a closure unless the opApply() uses 'scope'
+                    {
+                        fprintf(
```

Comment:
Won't that print a message regardless of any switch ? Having an uncategorized message you cannot disable when someone is using a feature doesn't seem like the best approach to me.